### ISSUE TITLE
Creating custom CSS theme throws error

### ISSUE THREAD

**Issue by javierluraschi** [rstudio/rstudio#4553]
Reprex:

```
rstudioapi::addTheme("https://raw.githubusercontent.com/javierluraschi/rstudio-pink/2d89dfaad30ff7bb139811a638949c607157534c/pink.rstheme", apply = TRUE)
```

Getting:

```
trying URL 'https://raw.githubusercontent.com/javierluraschi/rstudio-pink/master/pink.rstheme'
Content type 'text/plain; charset=utf-8' length 27560 bytes (26 KB)
==================================================
downloaded 26 KB

Error in value[[3L]](cond) : 
  Unable to add the theme file "/var/folders/ks/wm_bx4cn70s6h0r5vgqpsldm0000gn/T//Rtmp0c1Vca/pink.rstheme". The specified theme "pink" does not exist.
```

Tried removing `~/.R/rstudio/themes/` and restarting RStudio.

@MariaSemple

**Comment by MariaSemple** [comment]
I am able to reproduce this on build 1.2.1049.

It looks like the lack of theme name comment (e.g. `/* rs-theme-name: Pink */`) is causing this error on import, and preventing the removal of the theme. Additionally, it seems to be preventing re-adding a theme with the same file name (there is no pop-up requesting to replace the existing file, just an error that it cannot be replaced).

**Comment by jmcphers** [comment]
@javierluraschi How did you generate the `rstheme` file? (Do we have a bug somewhere that causes us to fail to write `rs-theme-name`?)

**Comment by javierluraschi** [comment]
I copy-pasted from the XCode theme... I was also able to reproduce with something even smaller, as in: https://github.com/javierluraschi/rstudio-pink/blob/09df79e2a67bca2604df64cbbd9cc85d07bd6e49/pink.rstheme

```
/* rs-theme-name: Pink */
/* rs-theme-is-dark: FALSE */
.ace_gutter {
  background: #ffbffd;
  color: #333
}
.ace_content {
  background: #ffe2f6;
}
```

It's also possible that I messed up something.

So try instead:

```
rstudioapi::addTheme("https://raw.githubusercontent.com/javierluraschi/rstudio-pink/09df79e2a67bca2604df64cbbd9cc85d07bd6e49/pink.rstheme", apply = TRUE)
```

**Comment by javierluraschi** [comment]
Oh wait! I'm missing a `;` in `color: #333`! I bet that was the issue, ugh.

**Comment by javierluraschi** [comment]
Oh, never mind, I'm missing `;` in the theme that works as well.

**Comment by MariaSemple** [comment]
I'm able to reproduce on 1.2.1335. The problem occurs when the theme is added via the rstudioapi or the UI.

**Comment by jmcphers** [comment]
Fixed via https://github.com/rstudio/rstudio/pull/4556.

**Comment by ronblum** [comment]
Verified fixed on 1.2.1522-1.
